cloud-deployment-guide/editor-plugin-version.md

```diff
@@ -92,7 +92,7 @@ Use the URL query parameters to specify the version of each premium plugin. This
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&tinycomments=2.0.0"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&tinycomments=2.0.0" referrerpolicy="origin"></script>
```

Comment:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?tinycomments=2.0.0" referrerpolicy="origin"></script>
```

Reply:
fixed:  http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-and-features/